features/user_can_use_the_navbar.feature

```diff
@@ -0,0 +1,16 @@
+Feature: User can see a sidebar
+  As a User
+  In order to be able to read and navigate the site
+  I would like to have content and navigating tools on the index
+
+  Scenario: User clicks register in navbar
+    Given I am on the "Index" page
+    And I should see "Dreamteam Publications"
+    When I click on "Register"
+    And I should be on the "Subscriber" page
+
+  Scenario: User clicks register in navbar
```

Comment:
Shouldn't this scenario be `User clicks login in navbar`?

Reply:
One other feature of Cucumber you can use to refactor your scenarios are [Scenario Outlines](https://cucumber.io/docs/reference#scenario-outline)

You could for instance re-write the scenarios in this feature as follow
```gherkin
Scenario Outline: User can visit registration/login page from navbar
  Given I am on the "Index" page
  Then I should see "Dreamteam Publications"
  When I click on <link>
  Then I should be on the <page> page

  Examples:
    | link     | page       |
    | Register | Subscriber |
    | Login    | Login      |
```